src/app/client/invite-employee/invite-employee.component.spec.ts

```
@@ -0,0 +1,28 @@
+  beforeEach(waitForAsync(() => {
+    TestBed.configureTestingModule({
+      imports: [InviteEmployeeComponent, NoopAnimationsModule],
+      providers: [provideHttpClient(), provideHttpClientTesting()],
+    }).compileComponents();
+  }));
+
+  beforeEach(() => {
+    fixture = TestBed.createComponent(InviteEmployeeComponent);
+    component = fixture.componentInstance;
+    fixture.detectChanges();
+  });
```

Biome

[error] Disallow duplicate setup and teardown hooks.

Disallow beforeEach duplicacy inside the describe function.

(lint/suspicious/noDuplicateTestHooks)